### ISSUE TITLE
FATAL TweakScale Error

### ISSUE THREAD

**Issue by InsanityBK04** [net-lisias-ksp/TweakScale#70]
I booted up my KSP and got a FATAL TweakScale Error
[KSP.log](https://github.com/net-lisias-ksp/TweakScale/files/3553608/KSP.log)
I cant find my Output_Txt.log file...
I did not get a screenshot sadly cause I closed my game and did not know i needed one. But it basically made a popup saying that there was a FATAL error and I needed to seek help and that my save game may be corrupted.

**Comment by Lisias** [comment]
Don't worry about the screenshot, the LOG has the information that was displayed on them.

I usually need the ModuleManager.ConfigCache for building fixes when needed, but on your case it's already a diagnosed problem:

```
[LOG 22:01:23.157] [TweakScale] WARNING: **FATAL** Found a showstopper problem on batteryBankMini (Z-200 Rechargeable Battery Bank).
[LOG 22:01:23.157] [TweakScale] ERROR: **FATAL** Part batteryBankMini (Z-200 Rechargeable Battery Bank) has a fatal problem due having duplicated properties - see issue [#34]( https://github.com/net-lisias-ksp/TweakScale/issues/34 ).
```

```
[LOG 19:58:41.202] Applying update RN_Soviet_Rockets/RN_Soviet_Rockets_TU/@PART[*]:NEEDS[TexturesUnlimited] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:02.817] Applying update TweakScale/patches/Squad/Squad_Util/@PART[batteryBankMini] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:08.616] Applying update FerramAerospaceResearch/_FARPartModule/@PART[*]:HAS[!MODULE[LaunchClamp]]:FOR[FerramAerospaceResearch] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:09.791] Applying update FerramAerospaceResearch/_FARPartModule/@PART[*]:HAS[!MODULE[FARWingAerodynamicModel],!MODULE[FARControllableSurface]]:AFTER[FerramAerospaceResearch] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:17.110] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[!MODULE[ModuleEngines*],!MODULE[ModuleHeatShield],~RSSROConfig[]]:BEFORE[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:17.224] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[~crashTolerance[>12]]:BEFORE[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:17.319] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[~gTolerance[<50]]:BEFORE[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:17.570] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[~maxPressure[<4000]]:BEFORE[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 19:59:17.637] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[~breakingForce[]]:BEFORE[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:00:11.592] Applying update RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Electrical/@PART[batteryBankMini]:FOR[RealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:00:46.464] Applying update RealismOverhaul/RO_SuggestedMods/RO_RCS_Config/@PART[*]:AFTER[RO-RCS] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:02:05.817] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[#RSSROConfig[*],!MODULE[ModuleAeroReentry]]:NEEDS[DeadlyReentry]:FOR[zzzRealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:02:07.042] Applying update RealismOverhaul/RO_RadialEnginesAttach/@PART[*]:HAS[#node_attach[*],#attachRules[1,0,1,1,0]]:FOR[zzzRealismOverhaul] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:02:10.880] Applying update RealismOverhaul/RealismOverhaul_Global_Config/@PART[*]:HAS[#RSSROConfig[*]]:FOR[zzzTagCleanup] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:02:11.184] Applying update B9PartSwitch/CopyEventsPropagator/@PART:FOR[zzzzzz-B9PartSwitch] to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
[LOG 20:02:16.190] Applying update Kopernicus/Config/SolarPanels/@PART:FINAL to Squad/Parts/Electrical/z-200Battery/z-200Battery.cfg/PART[batteryBankMini]
```

It's Realism Overhaul, already handled on forum on [this post](https://forum.kerbalspaceprogram.com/index.php?/topic/179030-14-tweakscale-under-lisias-management-2433-2019-0814/&do=findComment&comment=3629378).

In a nutshell, until RO guys fix their patches, download [this file](https://github.com/net-lisias-ksp/TweakScale/blob/dev/emergencial/2.4.3.4/Extras/TweakScale/HotFixes/RO-Stock_Electrical.cfg)] and save it on a folder inside your GameData (I suggest `GameData/__LOCAL/TweakScale/HotFixes`).

Let me know if this works for you.

**Comment by InsanityBK04** [comment]
I put the hotfix in D:\Steam\steamapps\common\Kerbal Space Program\GameData\TweakScale\patches
Is this the correct place?

**Comment by Lisias** [comment]
It will work, but it's not the best place. You will need to replace the file every time you update TweakScale.

I like to use `GameData/__LOCAL/TweakScale/HotFixes` because it's easy to remember and locate and very unlikely that anyone would delete it by accident.

**Comment by Lisias** [comment]
Did it worked fine? Can I close this issue?

**Comment by InsanityBK04** [comment]
oh yeah, it did work! I forgot to tell you.
